src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PedigreeAnnotation.java

```diff
@@ -101,4 +102,13 @@ void validateArguments(Collection<String> founderIds, GATKPath pedigreeFile) {
             logger.warn(this.getClass().getSimpleName() + " annotation will not be calculated, no 'founder-id' or 'pedigree' arguments provided");
         }
     }
+
+    /**
+     * This is a getter for the pedigree file, which could be null.
+     *
+     * @return The pedigree file
+     */
+    public @Nullable GATKPath getPedigreeFile() {
```

Comment:
@bbimber BTW for the future, for better or worse we generally don't use annotations like `Nullable`, although there are a handful of places throughout the source where they've crept in.

Reply:
OK. i was using it based on a handful of other places i saw it